timeClock.ino

```diff
@@ -154,7 +156,7 @@ void setup() {
   // Read Flash EEPROM, set colorSelect and projectSelect.
   colorSelect = EEPROM.read(1);
   projectSelect = EEPROM.read(2);
-  if (colorSelect == 0 && projectSelect == 0) { // Set to defaults if Flash EEPROM contains no data.
+  if (colorSelect == 255 && projectSelect == 255) { // Set to defaults if Flash EEPROM contains no data.
     colorSelect = 7;
```

Comment:
This was done since the EEPROM library changed and when setting the clock the memory gets set to 255 rather than 0.